### PR TITLE
Add about documentation button to docs home page

### PR DESCRIPTION
**Pull request by whitebeard10**
this PR is for #43249  
i have added the button and link 
image
have rephrased the words to make it appear in sync with other descriptions in the page

waiting for feedback or merge

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Add about documentation button to docs home page

We avoid referencing issues in PR titles; these become commit messages.

File: content/en/docs/home/_index.md [hunk 1]
```diff
@@ -21,47 +21,49 @@ description: >
...
+    button_path: "/training"
+  - name: Download
+    title: Download Kubernetes
+    description: Install Kubernetes or upgrade to the newest version.
+    button: "Download Kubernetes"
+    button_path: "/releases/download"
+  - name: about
+    title: About the docs
+    description: This website contains documentation for the current and previous 4 versions of Kubernetes.
+    button: "Available document versions"
```
Comment (sftim) [inline content/en/docs/home/_index.md:67]:
```suggestion
    button: "See available versions"
```

File: content/en/docs/home/_index.md
Comment (sftim) [inline content/en/docs/home/_index.md:1]: I'm not sure we need to make this change.

If we do, please only change the one card that we're editing; this makes code history easier to manage.

**Review by sftim — COMMENTED** [review]
Thanks; however, we shouldn't accept this change as it is.

File: content/en/docs/home/_index.md
Comment (whitebeard10) [reply content/en/docs/home/_index.md:1]: sure, thanks for the feedback

File: content/en/docs/home/_index.md [hunk 2]
```diff
@@ -21,47 +21,49 @@ description: >
...
+    description: "Get certified in Kubernetes and make your cloud native projects successful!"
+    button: "View training"
+    button_path: "/training"
+  - name: Download
+    title: Download Kubernetes
+    description: Install Kubernetes or upgrade to the newest version.
+    button: "Download Kubernetes"
+    button_path: "/releases/download"
+  - name: about
+    title: About the docs
```
Comment (Gauravpadam) [inline content/en/docs/home/_index.md:65]: I like "About the documentation" better

File: content/en/docs/home/_index.md (hunk 1 above)
Comment (Gauravpadam) [inline content/en/docs/home/_index.md:67]: (nit) I would say "See Available Versions"

**Review by Gauravpadam — COMMENTED** [review]
Thanks,

Here's some feedback from my end:

- We need not shorten the title of the card
- I'd try to use verbs for the button

**Comment by whitebeard10** [comment]
I have done the suggested changes, could you please check if it aligns with the suggestions.

**Review by Gauravpadam — COMMENTED** [review]
Just one concern,
Can we limit the changes to the targeted card?

**Comment by whitebeard10** [comment]
last change was me accidentally clicking sync fork, hope it doesn't cause any problem

**Comment by sftim** [comment]
/lgtm cancel
/hold

pending an update that addresses https://github.com/kubernetes/website/pull/43384/files#r1349970309

**Comment by niranjandarshann** [comment]
@whitebeard10 Could you please squash your commits: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by niranjandarshann** [comment]
> @niranjandarshann hey, can you suggest me what went wrong, i tried to squash as you suggested and i didn't change the files; however on previous pr it was deployed in preview by the bot but this time it failed.

@whitebeard10 you can restart netlify checks by force pushing an empty commit or rebasing against main. I think this work.

**Comment by sftim** [comment]
https://github.com/kubernetes/website/issues/43249 is now accepted.

Per https://github.com/kubernetes/website/issues/43249#issuecomment-1748202935, please use a verb for the button text.

**Comment by divya-mohan0209** [comment]
Hello @whitebeard10 , please could you advise if you're able to make the changes suggested in [this comment](https://github.com/kubernetes/website/pull/43384#issuecomment-1963016509)? Also, the PR needs to be rebased.

**Pull request closed without merging**